docs/baseline.md

```diff
@@ -59,6 +59,9 @@ Baseline status cannot or will not satisfy the following non-goals:
   Many Baseline features will not achieve 100% user reach soon or perhaps ever.
   If a web developer needs to support a globally uncommon or discontinued browser (e.g., Internet Explorer 11), then the developer needs to know the specific limitations of that browser, not a broad overview of developers’ most commonly required browsers.
   Baseline can’t be both.
+* **Identify support in assistive technology.**
+  Baseline should cover core accessibility support in browsers but it does not cover support in assistive tools (e.g., screen magnifiers, screen readers, alternative keyboards) that may be used along a browser to meet the requirements of people with disabilities.
```

Comment:
Suggested re-write since this is a non-goal (and an aspirational _should_ seems weird here):

"Baseline will not cover support in assistive technology (AT), which includes screen readers, screen magnifiers, voice control, and so on."